package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "discord-ytdl-core",
-    "version": "5.0.3",
+    "version": "5.0.4",
```

Comment:
please do not update the version

Reply:
Since the package can't be published to the same version, you rather change it yourself?
because I would love to see this feature in the npm as soon as I can...